bin/probot-run.js

```diff
@@ -10,7 +10,7 @@ const {findPrivateKey} = require('../lib/private-key')
 program
   .usage('[options] <apps...>')
   .option('-p, --port <n>', 'Port to start the server on', process.env.PORT || 3000)
-  .option('-t, --tunnel <subdomain>', 'Expose your local bot to the internet', process.env.SUBDOMAIN || process.env.NODE_ENV !== 'production')
+  .option('-t, --tunnel <subdomain>', 'Deprecated: localtunnel is no longer supported', process.env.SUBDOMAIN)
```

Comment:
Minor nit. Can we make this match the `console.warn` down below? I like the wording of "support has been removed" better than "no longer supported".